draft-httponlyprefix-weiss-http.md

```diff
@@ -68,29 +68,34 @@ that make sure that a cookie is not set on the client side using script.
 
 # Server Requirements
 
+These requirements apply to cookies received in a `Cookie` request header from the client.
+
 ## Cookie Name Prefixes
 
 ### The "__HttpOnly-" prefix
 
 If a cookie's name begins with a case-sensitive match for the string __HttpOnly-,
-then the cookie MUST also be:
+then this indicates that **all** the following are true:
```

Comment:
I think we want to have both:
* Requirements on when servers should send a Set-Cookie with an __HttpOnly prefix
* What can they make of Cookie request headers that have that prefix.

I just included the first and your wording only includes the latter :)

I'll land this and then tweak. Thanks!!